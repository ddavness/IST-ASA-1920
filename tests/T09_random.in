5,7
2
15
18
4
6
2 5
2 3
2 4
4 5
3 5
3 4
1 2
