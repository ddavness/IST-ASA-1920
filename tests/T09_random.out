18
18
18
6
6
